packages/bsky/src/util/retry.ts

```diff
@@ -1,38 +1,14 @@
-import { AxiosError } from 'axios'
-import { XRPCError, ResponseType } from '@atproto/xrpc'
-import { RetryOptions, retry } from '@atproto/common'
-import { Code, ConnectError } from '@connectrpc/connect'
+import { createRetryable } from '@atproto/common'
+import { ResponseType, XRPCError } from '@atproto/xrpc'
 
-export async function retryHttp<T>(
-  fn: () => Promise<T>,
-  opts: RetryOptions = {},
-): Promise<T> {
-  return retry(fn, { retryable: retryableHttp, ...opts })
-}
+export const RETRYABLE_HTTP_STATUS_CODES = [
+  408, 425, 429, 500, 502, 503, 504, 522, 524,
+]
```

Comment:
Was there a particular reason to move away from the `Set`?

Reply:
Ah I see now how it's used with `RetryAgent` 👍

Reply:
Still reverted this change as `set.has` is better than `array.includes` and `RetryAgent` is only instantiated once.